apps/docs/project.json

```diff
@@ -34,6 +34,22 @@
             "891377307993.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:prd",
             "891377307993.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:latest"
           ]
+        },
+        "uat": {
+          "push": true,
+          "build-args": ["DEPLOY_ENV=uat"],
+          "tags": [
+            "975050122570.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:uat",
+            "975050122570.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:latest"
+          ]
+        },
+        "staging": {
+          "push": true,
+          "build-args": ["DEPLOY_ENV=staging"],
```

Comment:
Should this be `sta` instead of `staging`?

@ianlnf

Reply:
staging